# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/NotificationUtils.cpp
# Commit: f9f3e107f36c
# Full Hash: f9f3e107f36c8cce438711ecef1b1a9574ae8f93
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2025-03-22 09:02:07
# Regressor Bug: 1948339
# File Overlap Count: 2
# Description:
#   Bug 1948339 - Part 2: Add NotificationObserver r=asuth
#   
#   This also makes page-bound notifications to not be stored via NotificationStorage.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D240642
# ==============================================================================

diff -r b84ed8a9da4c -r f9f3e107f36c dom/notification/NotificationUtils.cpp
--- a/dom/notification/NotificationUtils.cpp	Fri Mar 21 09:52:31 2025 +0000
+++ b/dom/notification/NotificationUtils.cpp	Fri Mar 21 09:52:31 2025 +0000
@@ -206,14 +206,11 @@
   return NS_ERROR_FAILURE;
 }
 
-void UnregisterNotification(nsIPrincipal* aPrincipal, const nsString& aId,
-                            CloseMode aCloseMode) {
+void UnregisterNotification(nsIPrincipal* aPrincipal, const nsString& aId) {
   // XXX: unpersist only when explicitly closed, bug 1095073
   UnpersistNotification(aPrincipal, aId);
   if (nsCOMPtr<nsIAlertsService> alertService = components::Alerts::Service()) {
-    alertService->CloseAlert(
-        aId,
-        /* aContextClosed */ aCloseMode == CloseMode::InactiveGlobal);
+    alertService->CloseAlert(aId, /* aContextClosed */ false);
   }
 }
 